Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -628,6 +629,19 @@ def bottle_formula(formula)
 
         return unless args.json?
 
+        if keg
+          keg_prefix = "#{keg}/"
+          path_exec_files = [keg/"bin", keg/"sbin"].select(&:exist?)
+                                                   .flat_map(&:children)
+                                                   .select(&:executable?)
+                                                   .map { |path| path.to_s.delete_prefix(keg_prefix) }
+                                                   .join(",")
```

Comment:
Since this is being serialised as JSON, should we not just keep it as an array?
```suggestion
                                                   .map { |path| path.to_s.delete_prefix(keg_prefix) }
```

Reply:
@carlocab Want to shrink the overhead on the manifest as much as possible here I think. I don't feel strongly, though, maybe we should just do that.

Reply:
Keeping it as an array will be much nicer for consuming with tools like `jq`. Doesn't make a huge difference for anything else.

I don't mind saying we don't care too much about `jq` users here tho.

Reply:
@carlocab applied this but needs to be turned back into a string when used as an annotation later